app/react/Library/Library.js

```diff
@@ -6,7 +6,6 @@ import LibraryModeToggleButtons from 'app/Library/components/LibraryModeToggleBu
 import requestState from 'app/Library/helpers/requestState';
 import LibraryLayout from 'app/Library/LibraryLayout';
 import { wrapDispatch } from 'app/Multireducer';
-import ImportProgress from 'app/Uploads/components/ImportProgress';
```

Comment:
@grafitto It seems like you brought some of the code from this component. Why do we need to replicate the code instead of reusing it?

Reply:
Reused